firestore-counter/functions/package.json

```diff
@@ -30,5 +30,8 @@
     "test": "mocha --require ts-node/register test/**/*.ts",
     "generate-readme": "firebase ext:info .. --markdown > README.md"
   },
+  "engines": {
```

Comment:
This is not how you set the runtime for an extension, you'll need to add a runtime field to extension.yaml. Please remove the "engines" field here, it gets ignored. 

Example: https://github.com/stripe/stripe-firebase-extensions/blob/next/firestore-stripe-subscriptions/extension.yaml#L65

Reply:
Also, please test all the extensions in Node 10 again since I'm assuming the previous testing you did was accidentally for Node 8.

Reply:
Thanks @laurenzlong 

PR updated!